python/lsst/analysis/tools/actions/plot/plotUtils.py

```diff
@@ -634,6 +635,9 @@ def plotProjectionWithBinning(
         extrema of the data).
     vmin, vmax : `float`, optional
         The min and max limits for the colorbar.
+    addExtremeScatter: `bool`, optional
+        Whether to include scatter points showing the positions of the
+        most extreme 15% head and tail of points?
```

Comment:
I'd drop the question mark and "Whether to". Points-> values? Also, subtly, it's the 15% most extreme, but one could read this as saying it's the head 15% _and_ the tail 15%.